deploy/test/test_cases/TEST_ID_18_helm_multiple_provider_multiple_secrets.sh

```diff
@@ -4,9 +4,10 @@ set -euxo pipefail
 # This test verifies that two Secrets Provider Jobs deploy successfully in the same namespace
 setup_helm_environment
 
+secret_value=$(openssl rand 10)
```

Comment:
i am guessing this is a binary secret? can you add a comment if this is correct?
also - if the SP succeeds to provide a binary secret, is it granted that it can a plaintext one?